### PR TITLE
Issue #21: Moving reference.views.inc into the views directory so that it will get included by views.

### PR DESCRIPTION
**Pull request by hosef**
Issue #21: Moving reference.views.inc into the views directory so that it will get included by views.

### CODE REVIEW

File: views/reference.views.inc [hunk 1]
```diff
@@ -85,3 +85,53 @@ function reference_field_views_data_views_data_alter(&$data, $field) {
...
 }
+
+/**
+ * Implements hook_views_plugins().
+ */
+function reference_views_plugins() {
+  $plugins = array(
+    'display' => array(
+      'reference' => array(
+        'title' => t('Entity Reference'),
```
Comment (jenlampton) [inline views/reference.views.inc]: These still refer to Entity reference.

**Comment by jenlampton** [comment]
I haven't had any problems with the file not getting picked up. Isn't any file ending in `.views.inc` in the module directory picked up by views? Usually the only code that goes into the `views` directory are PHP classes that are specific to views. Having the `*.views.inc` file in the root directory is also more useful (to humans) because it indicates that the module contains views integration at first glance. I don't think I'm in favor of this change.

**Comment by hosef** [comment]
> Isn't any file ending in `.views.inc` in the module directory picked up by views?

No, it only looks in the directory specified by your path in hook_views_api

**Comment by jenlampton** [comment]
This is ringing a bell. Is the module broken for you now without this change? I'll check my sites that are using reference views and see what state they are in too...

**Comment by hosef** [comment]
Yeah, I checked that it did not show up, and then moved the file and it worked.

**Comment by hosef** [comment]
https://github.com/backdrop-contrib/reference/issues/21#issuecomment-771976013

**Comment by jenlampton** [comment]
Fair enough.

**Pull request merged**